core/network/types/status.hpp

```diff
@@ -55,7 +51,8 @@ namespace kagome::network {
   template <class Stream,
             typename = std::enable_if_t<Stream::is_encoder_stream>>
   Stream &operator<<(Stream &s, const Status &v) {
-    return s << v.roles << v.best_number << v.best_hash << v.genesis_hash;
+    return s << v.roles << v.best_block.block_number << v.best_block.block_hash
```

Comment:
It might make sense to rename block_number and block_hash in BlockInfo to just number and hash.

Reply:
Absolutely agree. Need a third opinion.